sys/newlib/syscalls.c

```diff
@@ -90,6 +90,7 @@ void _exit(int n)
     while(1);
 }
 
+#ifndef DISABLE_HEAP_ALLOC
```

Comment:
This part of this file is a bit different in thew current RIOT.  Could you rebase? There is already a test for sbrk.